core/keyserver/epochs_test.go

```diff
@@ -173,15 +178,21 @@ func TestLowestSequenceNumber(t *testing.T) {
 		{"4", 0, 4, true},
 		{"4", 1, 4, true},
 		{"some_token", 0, 0, false},
-		{"", 1, 6, true},
+		{"", 2, 6, true},
```

Comment:
it's not totally obvious from github's diffs shown here why 1 became 2.
I can see that lowestSequenceNumber has changed a fair bit but without detailed poking about I'm not sure what's prompted 1 -> 2

Reply:
I'm not sure what the previous code was doing, but 2 seems like the correct number for the following reasons: 
- The test setup prepares two epochs:
   - Epoch 1 with a highest sequenced value of 6
   - Epoch 2 with a highest sequenced value of 10
- When asking for the mutations associated with epoch 2, and no `page_token`, we should start with 6 and go to 10.